tests/mail_testsuite/dummy_mail/smtp/dummy.py

```diff
@@ -1,16 +1,20 @@
 from mail import mail
 from mail.smtp import smtp
 
-def smtp_dummy(acnt, pswd):
+def smtp_dummy(acnt, pswd, attachments):
     frm = acnt
-    to = acnt
-    
-    Mail = mail.mail(smtp.smtp)
-    
-    Mail.login(acnt, pswd)
-    
-    Mail.send(frm, to, 'Dummy mail from USCC LAB', 'Hello members : )')
-    
-    Mail.quit()
+    to = [acnt]
+
+    myMail = mail.mail(smtp.smtp)
+    myMail.login(acnt, pswd)
+
+    attachments = []
+    if () != attachments:
```

Comment:
I don't understand why you compare it along with the assignment. That's a hard code.